logstash-core/lib/logstash/codecs/base.rb

```diff
@@ -11,7 +11,7 @@ module LogStash::Codecs; class Base < LogStash::Plugin
 
   def initialize(params={})
     super
-    config_init(@params)
+    config_init(@allow_env, @params)
```

Comment:
I am confused, where `@allow_env` is coming from, its not coming from the params in this method and is not declared any where?